samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication/app/src/utils/requiresInteraction.js

```diff
@@ -0,0 +1,11 @@
+export default function({ errorCode }) {
+  if (!errorCode || !errorCode.length) {
```

Comment:
```js
return !!((errorCode && errorCode.length) && (~errorCode... ||));
```

(I may be wrong on the logic.)

Reply:
Hmmm I agree the logic can be combined, but I think it makes the logic more complicated to understand.